pandas/io/tests/test_date_converters.py

```diff
@@ -138,6 +138,19 @@ def date_parser(date, time):
                                names=['datetime', 'prn']))
         assert_frame_equal(df, df_correct)
 
+    def test_parse_date_column_with_empty_string(self):
+        # GH 6428
+        data = """case,opdate
+                  7,10/18/2006
+                  7,10/18/2008
```

Comment:
yeah that's prob the best soln here (pls add to doc-string & a small warning/note section in io.rst if you don't mind).